src/install/f5/linux/install-tarball.mdx

```diff
@@ -0,0 +1,32 @@
+---
+componentType: default
+headingText: Download using tarball
+---
+
+    <Callout variant="tip">
+      We recommend [using your distribution package manager to install the infrastructure agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/install-infrastructure-linux-using-package-manager). If you choose to [manually install the agent](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux), you may also want to install the on-host integrations from the provided tarballs.
+    </Callout>
+
+    The manual install process is not automated. If you opt for manual install, you must place the different files in the correct folders, and ensure that the agent has all the permissions to execute the integrations.
+
+    To install an integration from a tarball:
+
+    1. Download the packaged integration file from the [tarball repository](https://download.newrelic.com/infrastructure_agent/binaries/linux/amd64/).
+    2. Unpack the tarball file according to our [integration file structure and placement rules](/docs/integrations/integrations-sdk/getting-started/integration-file-structure-activation), so the agent is able to find the definitions, configurations, and executables of the integration.
+    3. Place the binary that contains the definition file inside `newrelic-integrations` or `custom-integrations` in the [agent directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#agent-directory).
+    4. Place the integration's config file under the [plugin directory](/docs/infrastructure/install-configure-manage-infrastructure/linux-installation/tarball-manual-install-infrastructure-linux#configure-plugin).
+    5. Follow the JMX tool instructions below.
```

Comment:
I think we decided to change step 5. Check this: https://docs.newrelic.com/install/ibm-mq/?deployment=tar. Let me know if I should change it.